db/migrate/20171119094913_add_mtu_to_subnet.rb

```diff
@@ -0,0 +1,5 @@
+class AddMtuToSubnet < ActiveRecord::Migration
+  def change
+    add_column :subnets, :mtu, :integer, :default => 1500, :null => false, :limit => 8
```

Comment:
Should it be `:limit => 10` to allow for `4294967295` ?

Reply:
Limit here is the number of bytes used to store the integer

Reply:
@timogoebel all is fine on your side now ?

Reply:
@bagasse: Yes. Thanks. Can you add a test for this to make sure it really works on all supported databases?